source/getting-started.txt

```diff
@@ -0,0 +1,83 @@
+.. _relational-migrator-get-started:
+
+====================================
+Get Started with Relational Migrator
+====================================
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. meta::
+   :description: MongoDB Relational Migrator is a tool to help you migrate your relational workloads to MongoDB.
+   :keywords: database migration, database conversion
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Relational Migrator is a tool to help you migrate your relational 
+data to MongoDB. This page guides you through installing the tool 
+and running your first sync job.
+
+To get familiar with Relational Migrator capabilities, 
+features, and use cases review the follow resources:
+
+- :ref:`<supported-databases>`
+-  :ref:`<rm-migration-scenarios>`
+- `Relation Migrator Learning Byte <https://learn.mongodb.com/courses/mongodb-relational-migrator>`__
+
+Steps
+-----
+
+.. procedure::
+
+   .. step:: Install
+
+      You can install Relational Migrator with a
+      :ref:`single machine install <single-machine-install>` suitable 
+      for testing or a :ref:`unattended server install <unattended-server>` 
+      for production workloads.
+
+   .. step:: Create a project
+
+      Start using Relation Migrator by 
```

Comment:
```suggestion
      Start using Relational Migrator by 
```

Reply:
However, I'd suggest: 

"To start using Relational Migrator, create a project."

Reply:
Applied.